src/IB/IBFEPostProcessor.cpp

```diff
@@ -104,7 +104,7 @@ IBFEPostProcessor::registerScalarVariable(const std::string& name,
 {
     EquationSystems* equation_systems = d_fe_data_manager->getEquationSystems();
     System& system = equation_systems->add_system<System>(name + " reconstruction system");
-    system.add_variable(name, fe_order, fe_family);
+    if(!system.is_initialized()) system.add_variable(name, fe_order, fe_family);
```

Comment:
Maybe this should be `if (!is_from_restart)`?

You can find out if a run is from restart via:
```
    RestartManager* restart_manager = RestartManager::getManager();
    bool is_from_restart = restart_manager->isFromRestart();
```

Reply:
yes, this is cleaner.